.gitignore

```diff
@@ -3,3 +3,5 @@ spec/dummy
 .bundle
 
 *.log
+
+.project
```

Comment:
Thanks for the pointers to global gitignore. I've added one to my system and will see how to get rid of the change in this PR.  (Annoyed me too, but never enough to seek a solution. Sorry.)